.gitignore

```diff
@@ -20,6 +20,7 @@ node_modules
 packages/api/dist
 packages/api/@types
 packages/crdt/dist
+packages/desktop-client/src/locale/*_old.json
```

Comment:
the parser will generate *_old.json language files when it can no longer find the string in the source, i.e. the source string was removed or changed.

if we limit translations to automations and an external repository only, we can likely ignore this completely and disable the option in the init() call.